Phoenix/PHContext.h

```diff
@@ -10,6 +10,10 @@
 
 - (void) load;
 
+#pragma mark - Editing
+
+- (NSString *) primaryConfigurationPath;
```

Comment:
Thanks for the example - however seems like this will requires change to https://github.com/kasper/phoenix/blob/master/Phoenix/PHContext.m#L46. At that point I'm not sure what to do :)